src/styles/index.scss

```diff
@@ -1,3 +1,116 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  text-align: center;
+  align-items: flex-end;
+
+  width: 200px;
+  height: 408px;
+
+  border: 1px #f3f3f3 solid;
+  border-radius: 5px;
+
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position-y: 32px;
+  background-position-x: 19px;
+  background-color: $color-default;
```

Comment:
image it is part of content. better to make it as img element instead background

Reply:
In the task there is point "background-image: url() should be relative to the index.scss. So should start with ../images.". But I`m ok to rebuild with image element =)